### PR TITLE
Additional rewriting

### PR DESCRIPTION
**Pull request by Michael1993**
- Use the single sentence-per-line document style
- Reword/rewrite sentences to sound more natural
- Use simple words whenever possible

### CODE REVIEW

File: purejin/index.md [hunk 1]
```diff
@@ -98,19 +79,19 @@ trouble especially with large, complex applications.
 * use your list/set implementations instead of javas (is really easy)
 * binds different configurations without `if`s everywhere (built-in concepts)
 * binds in different scopes
-* use of your own scopes/lifecycles (is really easy)
-* use of your own provider interface to escape scoping problems (is really easy) 
+* uses your own scopes/lifecycles (is really easy)
+* uses your own provider interface to escape scoping problems (is really easy)
```
Comment (jbee) [inline purejin/index.md]: This is more confusing now, it does not use _your own_ out of the box but it is easy to setup _your own_.

**Review by jbee — APPROVED** [review]
Thanks @Michael1993, just one issue where I commented what it should express, maybe you have another idea how to rephrase it differently?

File: purejin/index.md (hunk 1 above)
Comment (Michael1993) [reply purejin/index.md]: Updated. What do you think?

Comment (jbee) [reply purejin/index.md]: Better, Thanks. I apply it to the scopes item as well.

File: purejin/index.md [hunk 2]
```diff
@@ -98,19 +79,19 @@ trouble especially with large, complex applications.
 * use your list/set implementations instead of javas (is really easy)
 * binds different configurations without `if`s everywhere (built-in concepts)
 * binds in different scopes
-* use of your own scopes/lifecycles (is really easy)
-* use of your own provider interface to escape scoping problems (is really easy) 
+* uses your own scopes/lifecycles (is really easy)
```
Comment (jbee) [inline purejin/index.md]:
```suggestion
* can use your own scopes/lifecycles (easy to setup)
```

**Comment by Michael1993** [comment]
Thanks. I truly believe this is an improvement.

**Comment by jbee** [comment]
@Michael1993 I know it isn't documented very accessible and assumes you know the ins and outs of the principles and reasons for doing dependency injection with containers. So if you have any more questions just let me know. From where I stand it is hard to write something down as purejin is very flexible and can be used in many styles so its hard to know where to start as it comes down to the individual circumstances and preferences.

**Pull request merged**